tamr_unify_client/project/attribute_configuration/resource.py

```diff
@@ -72,3 +82,123 @@ def __repr__(self):
             f"numeric_field_resolution={self.numeric_field_resolution!r}, "
             f"attribute_name={self.attribute_name!r})"
         )
+
+
+class AttributeConfigurationSpec:
+    """A representation of the server view of an attribute configuration."""
+
+    def __init__(self, client, data, api_path):
+        self._data = data
+        self.client = client
+        self.api_path = api_path
+
+    @staticmethod
+    def of(attribute_configuration):
+        """Creates an attribute configuration spec from an attribute configuration.
```

Comment:
The code itself is repeated, but quite short, which I think is fine. However, these docstrings are pretty long and repeated. Having to keep these in sync could become annoying. Maybe we should document how `*Spec` classes look (e.g. centralize the docstrings for `*Spec.of` into 1 place) and just reference that doc from the `*Spec` class docstring.

Then we would only need to document the `with_*` with type information.

Reply:
We should discuss this and come to a consensus that we document in the RFC #252